Duplicati/Library/Backend/OneDrive/MicrosoftGraphBackend.cs

```diff
@@ -491,33 +500,45 @@ protected T Get<T>(string url)
 
         private T SendRequest<T>(HttpMethod method, string url)
         {
-            var request = new HttpRequestMessage(method, url);
-            return this.SendRequest<T>(request);
+            using (var request = new HttpRequestMessage(method, url))
+            {
+                return this.SendRequest<T>(request);
+            }
         }
 
         private T SendRequest<T>(HttpMethod method, string url, T body) where T : class
         {
-            var request = new HttpRequestMessage(method, url);
-            if (body != null)
+            using (var request = new HttpRequestMessage(method, url))
+            using (request.Content = this.PrepareContent(body))
             {
-                request.Content = this.PrepareContent(body);
+                return this.SendRequest<T>(request);

```

Comment:
Is it now possible for `request.Content` to be `null`?  Does that have any side effects?

Reply:
It was previously possible for it to be null - the check was just made before calling Prepare Content then and now is done inside, so that the using block can capture the content to dispose it (it could be done in a try/finally, but using is smart enough to do a null check before attempting to dispose). There shouldn't be any change in behavior beyond that - this method is used both for requests with and without bodies (IIRC, this and the method it calls with are the two used for all requests for this backend).

Reply:
You're right.  Sounds good.